lms/envs/test_docker.py

```diff
@@ -1,567 +1,9 @@
-    },
-    "default": {
-        "university": "default_university",
-        "domain_prefix": "www",
-    }
-}
-
-MICROSITE_TEST_HOSTNAME = 'test-site.testserver'
-MICROSITE_LOGISTRATION_HOSTNAME = 'logistration.testserver'
-
-TEST_THEME = COMMON_ROOT / "test" / "test-theme"
-
-# add extra template directory for test-only templates
-MAKO_TEMPLATES['main'].extend([
-    COMMON_ROOT / 'test' / 'templates',
-    COMMON_ROOT / 'test' / 'test_sites',
-    REPO_ROOT / 'openedx' / 'core' / 'djangolib' / 'tests' / 'templates',
-])
-
-# Setting for the testing of Software Secure Result Callback
-VERIFY_STUDENT["SOFTWARE_SECURE"] = {
-    "API_ACCESS_KEY": "BBBBBBBBBBBBBBBBBBBB",
-    "API_SECRET_KEY": "CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC",
-}
-
-VIDEO_CDN_URL = {
-    'CN': 'http://api.xuetangx.com/edx/video?s3_url='
-}
-
-######### dashboard git log settings #########
-MONGODB_LOG = {
-    'host': MONGO_HOST,
-    'port': MONGO_PORT_NUM,
-    'user': 'edxapp',
-    'password': 'password',
-    'db': 'xlog',
-}
-
-NOTES_DISABLED_TABS = []
-
-# Enable EdxNotes for tests.
-FEATURES['ENABLE_EDXNOTES'] = True
-
-# Enable teams feature for tests.
-FEATURES['ENABLE_TEAMS'] = True
-
-# Enable courseware search for tests
-FEATURES['ENABLE_COURSEWARE_SEARCH'] = True
-
-# Enable dashboard search for tests
-FEATURES['ENABLE_DASHBOARD_SEARCH'] = True
-
-# Use MockSearchEngine as the search engine for test scenario
-SEARCH_ENGINE = "search.tests.mock_search_engine.MockSearchEngine"
-
-FACEBOOK_APP_SECRET = "Test"
-FACEBOOK_APP_ID = "Test"
-FACEBOOK_API_VERSION = "v2.2"
-
-######### custom courses #########
-INSTALLED_APPS += ('lms.djangoapps.ccx', 'openedx.core.djangoapps.ccxcon')
-FEATURES['CUSTOM_COURSES_EDX'] = True
-
-# Set dummy values for profile image settings.
-PROFILE_IMAGE_BACKEND = {
-    'class': 'storages.backends.overwrite.OverwriteStorage',
-    'options': {
-        'location': MEDIA_ROOT,
-        'base_url': 'http://example-storage.com/profile-images/',
-    },
-}
-PROFILE_IMAGE_DEFAULT_FILENAME = 'default'
-PROFILE_IMAGE_DEFAULT_FILE_EXTENSION = 'png'
-PROFILE_IMAGE_SECRET_KEY = 'secret'
-PROFILE_IMAGE_MAX_BYTES = 1024 * 1024
-PROFILE_IMAGE_MIN_BYTES = 100
-
-# Enable the LTI provider feature for testing
-FEATURES['ENABLE_LTI_PROVIDER'] = True
-INSTALLED_APPS += ('lti_provider',)
-AUTHENTICATION_BACKENDS += ('lti_provider.users.LtiBackend',)
-
-# ORGANIZATIONS
-FEATURES['ORGANIZATIONS_APP'] = True
-
-# Financial assistance page
-FEATURES['ENABLE_FINANCIAL_ASSISTANCE_FORM'] = True
-
-JWT_AUTH.update({
-    'JWT_SECRET_KEY': 'test-secret',
-    'JWT_ISSUER': 'https://test-provider/oauth2',
-    'JWT_AUDIENCE': 'test-key',
-})
-
-COURSE_CATALOG_API_URL = 'https://catalog.example.com/api/v1'
-
-COMPREHENSIVE_THEME_DIRS = [REPO_ROOT / "themes", REPO_ROOT / "common/test"]
-COMPREHENSIVE_THEME_LOCALE_PATHS = [REPO_ROOT / "themes/conf/locale", ]
-
-LMS_ROOT_URL = "http://localhost:8000"
+# noinspection PyUnresolvedReferences
+from .test import *
```

Comment:
I am replicating the existing behavior of the existing settings files. It makes no difference to me in this case.